Tests/swift-log-fileTests/swift_log_fileTests.swift

```diff
@@ -1,23 +1,21 @@
 import XCTest
 @testable import FileLogging
-import Logging
+@testable import Logging
 
 enum TestError: Error {
     case noDocumentDirectory
     case cannotGetFileSize
 }
 
-// TESTING ALERT:
-// This must be tested manually, single test by single test, due to "logging system can only be initialized once per process." in the swift logger.
-
 final class swift_log_fileTests: XCTestCase {
     let logFileName = "LogFile.txt"
     
     func testLogToFileUsingBootstrap() throws {
         let logFileURL = try getDocumentsDirectory().appendingPathComponent(logFileName)
         print("\(logFileURL)")
         let logFileHandler = try FileLogHandler(label: "Foobar", localFile: logFileURL)
-        LoggingSystem.bootstrap(logFileHandler.handler)
+        LoggingSystem.bootstrapInternal(logFileHandler.handler)
```

Comment:
this resolves the issue you mention in "testing alert"

packages _must_ be able to just pass a `swift test` invocation :)

Reply:
Nice! Thanks!